Source/View Controllers/TCCProfileViewController.swift

```diff
@@ -314,10 +328,21 @@ class TCCProfileViewController: NSViewController {
     override func prepare(for segue: NSStoryboardSegue, sender: Any?) {
         super.prepare(for: segue, sender: sender)
         guard let openVC = segue.destinationController as? OpenViewController else { return }
+        guard let window = self.view.window else {
+            return
+        }
         if let button = sender as? NSButton, button == addAppleEventButton {
             Model.shared.current = executablesAC.selectedObjects.first as? Executable
             openVC.completionBlock = {
-                $0.forEach { self.insetIntoAppleEvents($0) }
+                $0.forEach {
+                    switch $0 {
```

Comment:
With two uses of $0 right next to each other, lets use a `result in` for the second one so this line becomes `switch result {`.

Reply:
Yep, fixed! I also switched the references to self to weak.